src/managers/TileManager.ts

```diff
@@ -47,6 +48,25 @@ class TileManager {
         );
     }
 
+    public getStartingTiles(): Tile[] {
+        const session: Nullable<SessionDTO> = GeneralManager.getSession();
+        assert(session);
+        const tiles: Tile[] = [];
+        const random: seedrandom.PRNG = seedrandom(session.seed);
+        const veins: int[] = seededShuffle([9, 10, 10], session.seed); // 9 is gold, 10 are coal veins
+
+        for (const item of preplacedTiles) {
+            const tile: Tile = new Tile(random, item.type || veins.pop()!);
```

Comment:
FYI passing the random like this will have the effect that the 3 special tiles and the first 3 normal ones will have the same uuid. Not sure if it ever will be a problem.

Reply:
Oh damn, I didn't think of that. Great catch! I think it won't be a problem as the starting tiles don't get saved to the server, with that the id never gets used. You agree with this?